packages/mobile/src/exchange/ExchangeConfirmationCard.tsx

```diff
@@ -101,8 +101,8 @@ export default function ExchangeConfirmationCard(props: Props) {
             <TotalLineItem amount={totalAmount} />
           </View>
         </View>
-      </SafeAreaView>
-    </ScrollView>
+      </ScrollView>
+    </SafeAreaView>
```

Comment:
`SafeAreaView` should generally be inside `ScrollView` so it displays nicely on devices with a notch.
Any reason not to do that here?

Reply:
ah ha I got confused and thought it was the other way

Reply:
Looking at the code I see 2 places where SafeAreaView is inside the scroll view and  about 10+ where it is outside.

Reply:
Yes when I initially added support for iOS, I put all of them outside as a shortcut.
In the long term most of them are meant to be inside, except when there's a fixed button area at the bottom of the screen.